src/responses.js

```diff
@@ -0,0 +1,37 @@
+module.exports.start = ("Welcome to TechneBot! This bot will help you to facilitate various actions with Techne.\n\n"
+                        + "To view the list of rules for MakerSpace:\n/rules\n\n"
+                        + "To view the list of contacts for Techne's AY21/22 EXCO:\n/contacts\n\n"
+                        + "To get the Google Drive link to Techne's public archive of educational resources:\n/resources\n\n"
+                        + "To wake the bot up:\n/boop\n\n"
+                        + "For further queries regarding the bot or its usage, please contact @leehws. Thank you!");
```

Comment:
The parentheses aren't necessary since JavaScript is not line based! (This was only necessary for Python)